features/list.feature

```diff
@@ -0,0 +1,21 @@
+Feature: List
+  As a reader
+  I want to read posts from my favorite blogger
+  So that I can learn something
+  
+  Scenario: readers accessing published posts
+    Given I am on the post submission page again
```

Comment:
You can skip this step and start the scenario with `When I create a valid post`.